profiles/views.py

```diff
@@ -1,3 +1,24 @@
 from django.shortcuts import render
+from rest_framework import generics
+from profiles.models import StartUpProfile
+from profiles.serializers import StartUpProfileSerializer
+
+
+class StartUpProfilesView(generics.ListAPIView):  #pagination???
+    queryset = StartUpProfile.objects.all()
+
+    serializer_class = StartUpProfileSerializer
+
```

Comment:
To improve the user experience, it's common to allow ordering of the results in a list view. For example, allowing the startups to be ordered by creation date or name.
```
ordering_fields = ['name', 'created_at']
ordering = ['created_at'] 
```